dashboard/app/models/level.rb

```diff
@@ -69,7 +69,8 @@ def serializable_hash(options=nil)
   # Rails won't natively assign one-to-one association attributes for
   # us, even though we've specified accepts_nested_attributes_for above.
   # So, we must do it manually.
-  def assign_attributes(attributes)
+  def assign_attributes(new_attributes)
+    attributes = new_attributes.stringify_keys
```

Comment:
What is the effect of this change?  IIRC there may be some fields that aren't stored as strings in .level files.